server/tests/unit/proposal.unit.test.js

```diff
@@ -986,6 +1017,68 @@ describe('changeStatusProposalRequest', () => {
     await controllers.changeStatusProposalRequest(mockRequest, mockResponse);
 
     expect(mockResponse.status).toHaveBeenCalledWith(500);
-    expect(mockResponse.json).toHaveBeenCalledWith({ error: 'Some other error' });
+    expect(mockResponse.json).toHaveBeenCalledWith({
+      error: "Some other error",
+    });
+  });
+});
+describe("updateThesisRequestStatus", () => {
+  it("should handle correct status and return a success message", async () => {
+    const status = "Approve";
+
+    jest.spyOn(services, "updateThesisRequestStatus").mockResolvedValue();
+
+    const mockResponse = {
+      status: jest.fn().mockReturnThis(),
+      json: jest.fn(),
+    };
+
+    await controllers.updateThesisStatus(
+      { params: { proposalId: 115 }, body: { status } },
+      mockResponse
+    );
+
+    expect(mockResponse.status).toHaveBeenCalledWith(400);
+    expect(mockResponse.json).toHaveBeenCalledWith({
+      error: "Uncorrect id",
```

Comment:
Here the test says it should return a success message, but it's testing that it returns an error